api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:17-alpine
+
+WORKDIR /app
+
+COPY . .
+
```

Comment:
Out docker image now will get copy to . (that means) in api folder and for sencond (.) it is going in the /app directory
wihout WORKDIR we will have to specify /app